Brewfile

```diff
@@ -5,3 +5,4 @@ brew "ocrmypdf"
 brew "imagemagick"
 brew "qpdf"
 brew "ffmpeg"
+brew "xpdf"
```

Comment:
Seems like a shame to add yet another pdf-related dependency :/ but sounds like its worth avoiding the painful parsing with qpdf